pull.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash -ex
-VERSION=5.12.0
```

Comment:
Why don't we have these variables anymore? Now the version has to be changed three times when updating

Reply:
No it doesn't. If you read my commit message. I'm removing these variables because we're not using this package in cobia and we're not updating this anymore in bluefin.

Reply:
I read your commit message, but I still do not understand. This file has three occurrences of "5.12.0" now. Previously, there was only one. When 5.13.0 gets releases, we'll have to change three lines instead of one.

Reply:
We are not updating this package anymore in Bluefin and it will be removed in Cobia. This locks it in to this version so we can get through the bluefin release cycle...